packages/app-mobile/plugins/loadPlugins.ts

```diff
@@ -1,20 +1,25 @@
-
 import Setting from '@joplin/lib/models/Setting';
 import BasePluginRunner from '@joplin/lib/services/plugins/BasePluginRunner';
 import PluginService, { PluginSettings } from '@joplin/lib/services/plugins/PluginService';
 import PlatformImplementation from './PlatformImplementation';
 import { Store } from 'redux';
 import Logger from '@joplin/utils/Logger';
 import shim from '@joplin/lib/shim';
+import { AppState } from '../utils/types';
 
 const logger = Logger.create('loadPlugins');
 
 type CancelEvent = { cancelled: boolean };
 
-const loadPlugins = async (
-	// eslint-disable-next-line @typescript-eslint/no-explicit-any -- Old code before rule was applied
-	pluginRunner: BasePluginRunner, pluginSettings: PluginSettings, store: Store<any>, cancel: CancelEvent,
-) => {
+export interface Props {
+	pluginRunner: BasePluginRunner;
+	pluginSettings: PluginSettings;
+	store: Store<AppState>;
+	reloadAll: boolean;
+	cancelEvent: CancelEvent;
+}
+
+const loadPlugins = async ({ pluginRunner, pluginSettings, store, reloadAll, cancelEvent }: Props) => {
```

Comment:
Follow-up PR:
- [ ] Refactor `loadPlugins` and related logic into `lib/` for consistency.